pkg/controllers/provisioningv2/fleetcluster/fleetcluster.go

```diff
@@ -119,28 +132,33 @@ func (h *handler) createCluster(cluster *v1.Cluster, status v1.ClusterStatus) ([
 
 	agentNamespace := ""
 	clientSecret := status.ClientSecretName
+
+	objs := []runtime.Object{}
 	if mgmtCluster.Spec.Internal {
 		agentNamespace = fleetconst.ReleaseLocalNamespace
-		clientSecret = "local-cluster"
+		clientSecret = "local-kubeconfig"
```

Comment:
We probably don't need this anymore, the `status.ClientSecretName` for the `fleet-local/local` cluster should already be `local-kubeconfig`

Reply:
This is a nit though, I don't see harm in duplicating this